tests/src/FunctionalJavascript/CorporateContactFormTest.php

```diff
@@ -4,6 +4,7 @@
 
 namespace Drupal\Tests\oe_contact_forms\FunctionalJavascript;
 
+use Drupal;
```

Comment:
don't do this anywhere, not in tests, not in src, not anywhere, the drupal standard is to use backslash when using native classes, see https://www.drupal.org/project/drupal/issues/2053489#comment-7863665

Reply:
Yep, sorry. We were checking drupal-rector with @GilNovacomm and it was not working (no output). Setting "short imports" feature to true from rector.yml make it work and made this changes automatically. Don't worry, I take note and never used it again.